homepage/locales/vn.ts

```diff
@@ -0,0 +1,83 @@
+export default {
+  translation: {
+    common: {
+      openInBrowser: 'Mở trong trình duyệt',
+      downloadApp: 'Tải ứng dụng',
+      fileSystemBasedStorage: 'Lưu trữ trên file',
+      comingSoon: 'Sắp ra mắt',
+      webApp: 'Ứng dụng web',
+      desktopApp: 'Ứng dụng desktop',
+      mobileApp: 'Ứng dụng điện thoại',
+    },
+    header: {
+      forTeams: 'Dành cho teams',
+    },
+    hero: {
+      title: 'Tăng sự hài lòng, năng suất và sáng tạo.',
```

Comment:
IMO, this is more friendly
```suggestion
      title: 'Thêm vui vẻ, năng suất và sáng tạo.',
```

Reply:
Or using this verb `Cải thiện` might sound better 🤔

Reply:
Yea, @phiendp `Cải thiện` is definitely better than my translation but I think @trietphm is more friendly:) I'll use his one, unless you want to suggest a better one:)